### DISCUSSION TITLE
Δικαιώματα σε κείμενα, λεζάντες, βιογραφία, μελέτησε περίπτωσης

### DISCUSSION THREAD

**Discussion by ghost** [sdy60/syllabus#53] [Q&A] [answered]
@epidrome μπορούμε για να εξοικονομίσουμε χρόνο και να επεκτείνουμε τη λογική των φωτογραφιών και για τα κείμενα του χρονολογίου; Δηλαδή να είναι κείμενα ελεύθερα για χρήση π.χ. creative commons
Ευχαριστώ

**Comment by epidrome** [comment, marked as answer]
στις παιδαγωγικές σημειώσεις στο κάτω μέρος των αντίστοιχων οδηγιών γράφει:
https://courses-ionio.github.io/help/social/

> Το σκεπτικό σε αυτήν την δραστηριότητα είναι η μελέτη των αντίστοιχων κεφαλαίων του βιβλίου του μαθήματος σε βάθος και η ενεργή συμμετοχή στην παραγωγή απλού εκπαιδευτικού υλικού που απευθύνεται στους συμφοιτητές σας. Η δραστηριότητα συμμετοχικού περιεχομένου σκοπεύει στην καλύτερη κατανόηση και κυρίως στην απόκτηση μιας εποικοδομητικής στάσης απέναντι στην υπάρχουσα θεωρία, έτσι ώστε να μπορεί να (επανά)διατυπωθεί καθώς και να επεκταθεί. Η παραδοσιακή (και πλέον παρωχημένη) παιδαγωγική καθορίζει μια ύλη που κάποιος θα πρέπει να μάθει ως έχει, αλλά αυτή η προσέγγιση αντικαθίσταται μερικώς από μια πιο ενεργή αντιμετώπιση του διαθέσιμου μαθησιακού περιεχομένου, το οποίο θα πρέπει να συνδυαστεί με νέα ευρύματα και να αναδιοργανωθεί με τον μαθητή σε συμμετοχικό-παραγωγικό ρόλο και όχι ως απλό θεατή-καταναλωτή.
